### ISSUE TITLE
Feature request: add Twitter feed as a user on ZeroMe

### ISSUE THREAD

**Issue by crypdick** [HelloZeroNet/ZeroMe#31]
On Diaspora*, there are a few accounts you can follow which simply mirror popular Twitter feeds (which is a nice way of leaving Twitter for D* without missing out from pages you want to keep following). It would be nice to be able to to do something similar on ZeroMe.

**Comment by kseistrup** [comment]
Even feeding RSS/Atom feeds directly into ZeroNet would be a boon.

**Comment by HelloZeroNet** [comment]
Good idea, and you can do it right now: you just need to write a bot that is read the feed/twitter, then add it to the .json file and publish to the network.
Here is the sourcecode for the "P2P news flow" @ ZeroTalk: https://gist.github.com/HelloZeroNet/006347ed183e7f3721ff

**Comment by crypdick** [comment]
Awesome @HelloZeroNet. So, I imagined creating a new ZeroID for each unique feed as well as a unique ZeroMe profile for each. I can't seem to make another ZeroID, though. How should I proceed?

**Comment by kustomzone** [comment]
Someone has posted instructions on ZeroTalk for making a ZeroMe user profile feed bot.

http://127.0.0.1:43110/Talk.ZeroNetwork.bit/?Topic:3_1AWwhg4EiWAVttfQboJZ4wJfX3WawfJT3h/ZeroMe+Feed+Bot
